src/main/java/service/impl/FruitOperationPerformer.java

```diff
@@ -0,0 +1,38 @@
+package service.impl;
+
+import java.util.List;
+import model.FruitTransaction;
+import service.Performer;
+import strategy.TransactionStrategy;
+import strategy.operation.OperationHandler;
+
+public class FruitOperationPerformer implements Performer<FruitTransaction> {
+    private final TransactionStrategy transactionStrategy;
+
+    public FruitOperationPerformer(TransactionStrategy transactionStrategy) {
+        this.transactionStrategy = transactionStrategy;
+    }
+
+    @Override
+    public void performProcess(FruitTransaction transaction) {
+        OperationHandler handler = transactionStrategy.get(transaction.getOperation());
+        handler.doOperation(transaction.getFruit(), transaction.getQuantity());
+    }
```

Comment:
unused as of now
make it a private method, call it inside of a public method, there's some code duplication